src/components/contactUs.css

```diff
@@ -0,0 +1,49 @@
+input[type='text'],
+textarea,
```

Comment:
make the text areas active status either change or hide (the blue border around the text zones)